tutorials/notebooks/keras/ptq/example_keras_activation_threshold.ipynb

```diff
@@ -0,0 +1,866 @@
+        "]\n",
+        "\n",
+        "# If you are curious you can add any of the below quantisation methods as well.\n",
+        "#QuantizationErrorMethod.MAE\n",
+        "#QuantizationErrorMethod.KL\n",
+        "#QuantizationErrorMethod.LP\n",
+        "\n",
+        "# Iterate and build the QuantizationConfig objects\n",
+        "for error_method in error_methods:\n",
+        "    q_config = mct.QuantizationConfig(\n",
+        "        activation_error_method=error_method,\n",
+        "        weights_error_method=error_method,\n",
+        "        weights_bias_correction=weights_bias_correction,\n",
+        "        shift_negative_activation_correction=shift_negative_activation_correction,\n",
+        "        z_threshold=z_threshold\n",
+        "    )\n",
+        "\n",
+        "    q_configs_dict[error_method] = q_config"
+      ],
+      "metadata": {
+        "id": "jtiZzXmTjxuI"
+      },
+      "id": "jtiZzXmTjxuI",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Finally we quantise the model, this can take some time."
+      ],
+      "metadata": {
+        "id": "8W3Dcn0jkJOH"
+      },
+      "id": "8W3Dcn0jkJOH"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ba0c6e55-d474-4dc3-9a43-44b736635998",
+      "metadata": {
+        "id": "ba0c6e55-d474-4dc3-9a43-44b736635998"
+      },
+      "outputs": [],
+      "source": [
+        "quantized_models_dict = {}\n",
+        "\n",
+        "for error_method, q_config in q_configs_dict.items():\n",
+        "    # Create a CoreConfig object with the current quantization configuration\n",
+        "    ptq_config = mct.core.CoreConfig(quantization_config=q_config)\n",
+        "\n",
+        "    # Perform MCT post-training quantization\n",
+        "    quantized_model, quantization_info = mct.ptq.keras_post_training_quantization_experimental(\n",
+        "        in_model=float_model,\n",
+        "        representative_data_gen=representative_dataset_gen,\n",
+        "        core_config=ptq_config,\n",
+        "        target_platform_capabilities=tpc\n",
+        "    )\n",
+        "\n",
+        "    # Update the dictionary to include the quantized model\n",
+        "    quantized_models_dict[error_method] = {\n",
+        "        \"quantization_config\": q_config,\n",
+        "        \"quantized_model\": quantized_model,\n",
+        "        \"quantization_info\": quantization_info\n",
+        "    }\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Threshold and Distribution Visulisation"
+      ],
+      "metadata": {
+        "id": "A8UHRsh2khM4"
+      },
+      "id": "A8UHRsh2khM4"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "To assist with understanding we will now plot the threshold and distributuion from two of Mobilenetv2's layers.\n",
```

Comment:
This paragraph needs to include more explanation regarding what we're going to see here, how it is affected by the threshold selection method and why it is helpful.